run.py

```
@@ -0,0 +1,501 @@
+async def profile(callback: CallbackQuery):
+    """Профиль пользователя"""
+    user_id = callback.from_user.id
+    user_info = await db.get_user(user_id)
+    
+    if not user_info:
+        await callback.answer("❌ Пользователь не найден!")
+        return
+    
+    bots = await db.get_user_bots(user_id)
+    
+    text = f"👤 Ваш профиль\n\n"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)